app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CreateEventActivity.java

```diff
@@ -182,67 +188,80 @@ private void updateDateViews() {
     }
 
     private void editEvent() {
-        final String eventNr = getIntent().getExtras()
-                                          .getString(Const.EVENT_NR);
-        TUMOnlineRequest<DeleteEvent> request = new TUMOnlineRequest<>(
-                TUMOnlineConst.Companion.getDELETE_EVENT(), this, true);
-        request.setParameter(Const.EVENT_NR, eventNr);
-        request.fetchInteractive(this, new TUMOnlineRequestFetchListener<DeleteEvent>() {
-            @Override
-            public void onNoInternetError() {
-                showErrorDialog(getString(R.string.no_internet_connection));
-            }
+        final String eventId = getIntent().getStringExtra(Const.EVENT_NR);
+        if (eventId == null) {
+            return;
+        }
 
-            @Override
-            public void onFetch(DeleteEvent response) {
-                Utils.log("Event successfully deleted (now creating the edited version)");
-                TcaDb.getInstance(getApplicationContext())
-                     .calendarDao()
-                     .delete(eventNr);
-                createEvent();
-            }
+        // Because we don't show a loading screen for the delete request (only for the create
+        // request), we use a short Toast to let the user know that something is happening.
+        Toast.makeText(this, R.string.updating_event, Toast.LENGTH_SHORT).show();
```

Comment:
Meh.
We should probably tackled this in another PR and show a snackbar

Reply:
I agree. This is a quick fix for now, because we did not display any information before. 

I want to rethink how we show download statuses generally at some point. For instance, imagine we have a cached response that we display to the user. If the user decides to refresh and an exception occurs, we currently show a full-screen error. However, ideally, we would display it as a `Snackbar` or `AlertDialog` (depending on ceverity).